### DISCUSSION TITLE
Is there any way to select  specific path of url for filename?

### DISCUSSION THREAD

**Discussion by snwefly** [gildas-lormeau/SingleFile#1204] [Q&A]
Test url

https://old.reddit.com/r/MMORPG/comments/159t4sl/cleaning_my_room_and_look_what_i_found/

Is there any way to select  "159t4sl" part as file name?

Thanks

**Comment by gildas-lormeau** [comment]
You should be able to get it via `{url-last-segment}`

**Reply by snwefly** [reply]
I tired that and the result is  "cleaning_my_room_and_look_what_i_found.html"

Is it possible to have template something like "url-segment-x"  ?
